content/doc/book/using/using-agents.adoc

```diff
@@ -196,3 +196,26 @@ Building remotely on agent1 in workspace /home/jenkins/workspace/First Job to Ag
 agent1
 Finished: SUCCESS
 ----
+
+=== Launching agent from console
+
+agent.jar is meant to be launched by Jenkins, typically through a remote
+shell like ssh/rsh. +
+Jenkins controller then communicates with this agent through stdin and
+stdout.
```

Comment:
We generally prefer sentence per line and don't want to use continuation characters to insert line breaks.  I removed the reference to `rsh` because it is [insecure](https://www.informit.com/articles/article.aspx?p=169465) and is not available on many distributions.

```suggestion
The `agent.jar` is meant to be launched by Jenkins, typically through a remote shell like ssh.
The Jenkins controller communicates with this agent through stdin and stdout.
```